flow/scripts/open.tcl

```diff
@@ -63,9 +63,3 @@ if {[env_var_equals GUI_TIMING 1]} {
 }
 
 fast_route
-
-if {[env_var_equals GUI_SHOW 1]} {
```

Comment:
@maliberty keep this. It is useful to minimize the GUI and have it unminimized when it is ready.

Reply:
We have discussed how that doesn't work in wayland or gnome.

Reply:
From https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/2600

```
The simplest solution is the oldest, just show the window from the start and wait for things to finish loading. 
```
to which you agreed.

Reply:
> We have discussed how that doesn't work in wayland or gnome.

If the window is manually minimized, then I think the unminimize will work. I will give it a go after this is merged.

Reply:
My guess is it won't work if the user action is too far removed from the unminimize in time.